server/index.js

```diff
@@ -32,14 +32,22 @@ app.get('/api/gardenStats', (req, res, next) => {
 });
 
 app.get('/api/plantsInGarden/:plantId', (req, res, next) => {
+  const plantId = req.params.plantId;
   const sql = `
   select *
     from "plantsInGarden"
   where "plantId" = $1
   `;
-  const params = [req.body.plantId];
+  const params = [plantId];
   db.query(sql, params)
     .then(result => {
+      if (!result.rows[0]) {
+        res.status(200).json({
```

Comment:
If there is an error, then `200` is not a good status code (OK). Instead a `404` would make more sense if something is Not Found.